src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpThurloeDAO.scala

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.dsde.firecloud.dataaccess
+
+import akka.actor.ActorSystem
+import org.broadinstitute.dsde.firecloud.FireCloudConfig
+import org.broadinstitute.dsde.firecloud.model.{ThurloeNotification, Notification}
+import org.broadinstitute.dsde.firecloud.service.UserService
+import org.broadinstitute.dsde.firecloud.utils.RestJsonClient
+import spray.http.{OAuth2BearerToken, HttpHeaders, HttpResponse, HttpRequest}
+import spray.httpx.TransformerPipelineSupport.WithTransformerConcatenation
+import spray.client.pipelining._
+import spray.json.DefaultJsonProtocol._
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
+import org.broadinstitute.dsde.firecloud.model._
+import spray.httpx.SprayJsonSupport._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/**
+ * Created by mbemis on 10/21/16.
+ */
+class HttpThurloeDAO ( implicit val system: ActorSystem, implicit val executionContext: ExecutionContext )
+  extends ThurloeDAO with RestJsonClient {
+
+  val fireCloudHeader = HttpHeaders.RawHeader("X-FireCloud-Id", FireCloudConfig.FireCloud.fireCloudId)
+  val adminToken = HttpGoogleServicesDAO.getAdminUserAccessToken
+
+  override def sendNotifications(notifications: List[Notification]): Future[Boolean] = {
```

Comment:
And why is this returning Boolean? Seems kind of silly especially since the value is never checked. It should throw an exception on failure or return a Try.